packages/atomic/tailwind.config.js

```diff
@@ -40,6 +40,8 @@ module.exports = {
         sm: 'var(--atomic-text-sm)',
         base: 'var(--atomic-text-base)',
         lg: 'var(--atomic-text-lg)',
+        xl: 'var(--atomic-text-xl)',
+        '2xl': 'var(--atomic-text-2xl)',
       },
       screens: {
```

Comment:
I believe it would be a good idea to update the screen widths here to match with the one in v1.html (1023 and 1024).

Reply:
https://tailwindcss.com/docs/breakpoints I followed Tailwinds default breakpoints (1024 lg, 640 sm)

Reply:
yes, I mean we should update `mobile-only` and `desktop-only` to be one below `lg` and equal to `lg`, so that every layout change happens at the same point.